### DISCUSSION TITLE
How do I get RuneKit to show up in the action bar on MacOS Sononma or higher?

### DISCUSSION THREAD

**Discussion by jlczuk** [whs/runekit#42] [Q&A macOS] [answered]
Since MacOS Sonoma, I have not been able to use RuneKit.  I have it fully configured with the proper permissions.  I see it downloading all the apps.  But it does not who up in the top bar when Runescape is running.

**Comment by jlczuk** [comment, marked as answer]
I found the answer in this image copied from ImgUR.  The issue is related to Apple changing the screen settings.  I could also be related to the fact that I'm on a new M2 MacBook Pro, which likely has different screen defaults.

Selecting a different default resolution fixed the problem for me.

![image](https://github.com/whs/runekit/assets/8922067/7d2be84e-5190-4aac-82e8-e8dc3df6e50d)

**Reply by JacobvanP** [reply]
The issue was related to RuneScape changing the display settings. I haven't found a proper solution yet sadly besides what I wrote in the screenshot. (Also on the M2 one)
